src/Server/types.tsx

```diff
@@ -0,0 +1,40 @@
+export type fetchListProps = {
+  lat: number;
+  lng: number;
+  dist: number;
+  rad: number;
+  type: string;
+};
+export type TankResult = {
+  stations: {
+    id: string;
+    brand: string;
+    street: string;
+    lat: number;
+    lng: number;
+    dist: number;
+    diesel: number;
+    e5: number;
+    e10: number;
+    houseNumber: string;
+    postcode: number;
+  }[];
+};
+export type All = {
+  stations: {
+    id: string;
+    name: string;
+    brand: string;
+    street: string;
+    place: string;
+    lat: number;
+    lng: number;
+    dist: number;
+    diesel: number;
+    e5: number;
+    e10: number;
+    isOpen: boolean;
+    houseNumber: string;
+    postcode: number;
+  }[];
+};
```

Comment:
You could make a type for station, apply it to stations like `stations: station[]` and extend your station type to match your type definition of `All`. 

Short note - I would suggest a more descriptive name for the type `All`

And more out of curiosity - TankResult sounds a little bit like a mixture of German & englisch probably (Re-)fuelResult could work here? (But this is more due to personal opinion)